services/packagist/packagist-license.service.js

```diff
@@ -59,17 +58,26 @@ module.exports = class PackagistLicense extends BasePackagistService {
   }
 
   transform({ json, user, repo }) {
-    const branch = this.getDefaultBranch(json, user, repo)
```

Comment:
We can delete `getDefaultBranch()` from the base class now. It is not being called anywhere.